src/main/content/_assets/js/builds.js

```diff
@@ -1002,6 +1002,23 @@ $(document).ready(function () {
                         document.body.removeChild(anchor);
                         url.revokeObjectURL(anchor.href);
                     }, 1);
+                    var genProjModal = document.getElementById("generate-project-modal");
+                    genProjModal.style.display = "block";
```

Comment:
We should avoid inline styles to practice the separation of styles (CSS) from the layouts (HTML).  Additionally, inline styles ignore the cascading stylesheets behavior and can add complexity to how styles are applied to HTML elements.

A suggestion is to have the modal be hidden by default through `.generate-project-modal-class`  and apply a "is visible" class on the modal that takes priority over the default hidden styling when it needs to be presented to the user. The is visible class can then be reused for future modals and unify how modals are made visible on the website.  

Here is how vanilla JS Carbon controls a modal https://the-carbon-components.netlify.app/component/modal--default.html

Reply:
ok here the modal popup is hidden by default. I will change this inline style by adding class for show and hide popup

Reply:
committed changes - removed inline style in js

Reply:
Thank you for putting in the extra work to rework the inline styling